apps/condo/domains/acquiring/schema/MultiPayment.js

```diff
@@ -56,8 +58,13 @@ const { RESIDENT, STAFF } = require('@condo/domains/user/constants/common')
 
 const { ACQUIRING_INTEGRATION_FIELD } = require('./fields/relations')
 
-
-
+const ERRORS = {
+    NOT_UNIQUE_INVOICES: {
+        code: BAD_USER_INPUT,
+        type: MULTIPAYMENT_NOT_UNIQUE_INVOICES,
+        message: 'Not unique invoices',
```

Comment:
There are some other fields, that would be great to define here. Eg `messageForUser` and `fields`

Reply:
This error is for API. Residents can't check the invoice twice within the interface.